plugins/destination/file/client/read.go

```diff
@@ -19,6 +19,11 @@ func (c *Client) Read(_ context.Context, table *schema.Table, res chan<- arrow.R
 		return fmt.Errorf("reading is not supported when `path` contains UUID variable. Table: %q", table.Name)
 	}
 	name := c.spec.ReplacePathVariables(table.Name, uuid.NewString(), time.Time{})
+
+	if syncAfterWrite {
+		time.Sleep(500 * time.Millisecond)
```

Comment:
I've used `assert.EventuallyWithT` in the cloud storage destinations (gcs/azblob/S3) but here we use the `TestInsert` functionality from the SDK (`testPlugin` - except for `parquet` type or compressed formats) and it needs to be able to see the rows immediately after the write call. For parquet or compressed formats, we use `testPluginCustom` which follows the `write / close / read` pattern.